charters/DataOperations/charter.md

```diff
@@ -0,0 +1,101 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team (DataOps) is a user-focused group aiming to connect the scientific community to the DCP through the effective tools and resources generated by the DCP which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution. These products include data, metadata, analysis tools, data access features. DataOps ensures that data flows through the DCP end-to-end in an efficient manner, is appropriately validated, and fulfills the needs and expectations of data consumers. DataOps maintains a DCP-wide vision through their internal focus on end-to-end data flow, as well as their community-focused goal of conveying the value of the entire DCP.
+
+## Definitions
+
+*Data contribution*: a set of raw data derived from the same experimental process, and the metadata describing that process
+
+*Release* (verb): to make metadata and data openly accessible
+
+*Data distribution*: the packaging of metadata and data from multiple data contributions in meaningful ways, along with relevant analysis pipelines, possible integrative analysis results, user vignettes, access features, and any announcements or documentation supporting it
+
+*HCA Snapshot*: a type of data distribution that occurs on a regular cadence containing all HCA data currently available that meets predetermined criteria, such as quality standards
+
+*Data workflow*: the sequence of tasks executed by the DCP for each data contribution beginning with first notification of a dataset (typically contact with the data generator or identification of data in a data archive or in scientific literature) all the way through data analysis by the DCP and continued through usability by data consumers
```

Comment:
I would prefer we not introduce this term "data workflow" as given here; workflow is already a thing and this will be confusing. Could there be another term that is more discriminant in conversation that the teams can internalize?

Reply:
Subbed in "data lifecycle" in 1ea796c. Hopefully that clarifies things.

Reply:
This has been quiet, so I will resolve